packages/grid/_modules_/grid/components/DefaultToolbar.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useGridSelector } from '../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../hooks/utils/useOptionsProp';
+import { ApiContext } from './api-context';
+import { GridToolbar } from './styled-wrappers/GridToolbar';
+
+// TODO: We can list separate features here, eg. column picker
+export interface DefaultToolbarProps {}
```

Comment:
Not sure I understand the purpose of this interface

Reply:
My initial idea was to list the separate components here and then add logic if there are provided from outside to overwrite the ones we provide as default but now I don't think it is needed since developers can provide an entire custom Toolbar